azure-storage-js-v10-quickstart-v4/README.md

```diff
@@ -0,0 +1,62 @@
+---
+page_type: sample
```

Comment:
Please merge the v3 and v4 readmes into the same file.  Maintaining two files for each sample is going to be a lot more work to maintain and I do not see enough differences between the two to support that maintenance overhead.